src/redirect/server.js

```diff
@@ -154,5 +152,10 @@ app.get('/:file.json', async (req, res) => {
     res.status(201);
```

Comment:
The filePath on line 148 needs top be updated now that the example files are in the `/fixtures` directory.

Reply:
Fixed!